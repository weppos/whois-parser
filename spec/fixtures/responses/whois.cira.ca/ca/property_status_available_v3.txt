Not found: u34jedzcq.ca

%
% Use of CIRA's WHOIS service is governed by the Terms of Use in its Legal
% Notice, available at http://www.cira.ca/legal-notice/?lang=en
%
% (c) 2019 Canadian Internet Registration Authority, (http://www.cira.ca/)
