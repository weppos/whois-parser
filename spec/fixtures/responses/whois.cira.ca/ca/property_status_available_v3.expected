#status
  %s == :available

#available?
  %s == true

#registered?
  %s == false
